settings.yaml

```diff
@@ -37,6 +37,19 @@ ExternalApi:
         settings-key: "QtumApiService-QtumInsightApi"
         types:
             - Tag
+    DirectNodeApi:
+        Url:
+            settings-key: "QtumApiService-DirectNodeApiUrl"
+            types:
+                - Tag
+        UserName:
+            settings-key: "QtumApiService-DirectNodeApiUserName"
+            types:
+                - Tag
+        Password:
+            settings-key: "QtumApiService-DirectNodeApiPassword"
```

Comment:
Add "Secret" type for this field. Look at confluence for information about that tag